### DISCUSSION TITLE
keras_cv.losses.IoULoss documentation not found

### DISCUSSION THREAD

**Discussion by cyildiz** [keras-team/keras-hub#1995] [Q&A]
Hi, 
I cannot reach the documentation for the loss function `keras_cv.losses.IoULoss()`. The Links found using Google search no longer work. Will this method and its documentation be available through Keras Hub eventually?

Currently it is not listed anywhere on the https://keras.io/keras_hub/ or https://keras.io/api/.
